src/libs/ck-libs/ampi/ampiOneSided.C

```diff
@@ -735,6 +738,28 @@ AMPI_API_IMPL(int, MPI_Win_create, void *base, MPI_Aint size, int disp_unit,
   return MPI_SUCCESS;
 }
 
+AMPI_API_IMPL(int, MPI_Win_allocate, MPI_Aint size, int disp_unit, MPI_Info info, MPI_Comm comm, void *baseptr, MPI_Win *win)
+{
+  AMPI_API("AMPI_Win_allocate");
+
+  int res = MPI_Alloc_mem(size, info, (void**)baseptr);
+  if(res != MPI_SUCCESS)
+    return res;
```

Comment:
Should call ampiErrhandler() instead of always returning the error code here

Reply:
done.